src/tracks/queue.rs

```diff
@@ -278,6 +278,13 @@ impl TrackQueue {
 
         inner.stop_current()
     }
+
+    /// Returns Vec of all handles in queue
+    pub fn to_vec(&self) -> Vec<TrackHandle> {
```

Comment:
```suggestion
    pub fn current_queue(&self) -> Vec<TrackHandle> {
```

This is a little more descriptive of the function's purpose here.